src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { useState } from "react";
+
+const LoginView = ({ onLoggedIn }) => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    const data = {
+      Username: username,
+      Password: password
+    }
+
+    fetch('https://render-movie-api.onrender.com/login', {
+      method: 'POST',
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(data)
+    }).then((response) => response.json())
+      .then((data) => {
+        console.log('Login response: ', data);
```

Comment:
When you get to deploy this application, you should be mindful of information is being logged since logging a token exposes it. You can use the logs to debug, but remove them when you ship your code.